.codespell.ignore

```diff
@@ -0,0 +1,2 @@
+iff
+create
```

Comment:
There's a mismatch between this file, which has `create`, and `.pre-commit-config.yaml` which lists `creat` in the ignore list argument:

https://github.com/ParaToolsInc/taucmdr/blob/63f912bcbf7dc15c83d308aba2d3d369026571f1/.pre-commit-config.yaml#L62

Should these match?

Reply:
actually, I meant to delete .codespell.ignore

Reply:
In that case, once `.codespell.ignore` is deleted, I think this PR is ready to merge.